### ISSUE TITLE
[Bug]: Attribute Error For Empty Datasets

### ISSUE THREAD

**Issue by haxdds** [alpacahq/alpaca-py#317]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When there is data missing in the response of a data request as described in 

https://github.com/alpacahq/alpaca-py/issues/230

and 

https://github.com/alpacahq/alpaca-py/issues/282

An attribute error is thrown.

### Expected Behavior

The error should be handled and the user should receive an empty response.

### SDK Version I encountered this issue in

0.8.2

### Steps To Reproduce

```markdown
https://github.com/alpacahq/alpaca-py/issues/230
https://github.com/alpacahq/alpaca-py/issues/282
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

**Comment by markolonius** [comment]
I have been getting this error message as well that seems to be same problem:

`python3.11/site-packages/alpaca/data/models/bars.py", line 47, in __init__
    BAR_MAPPING[key]: val for key, val in raw_data.items() if key in BAR_MAPPING
                                          ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'items'
`

any updates or work arounds?

**Comment by hiohiohio** [comment]
Please let me handle this issue.

**Comment by hiohiohio** [comment]
realized that we need to design handling empty data in wider area. It may take a bit of time to fix codebase.
as a workaround, can you please consider to use `raw_data=True` parameter?

```
client = StockHistoricalDataClient(api_key=api_key, secret_key=secret_key, raw_data=True)
```

**Comment by hiohiohio** [comment]
Released v0.14.0. Please try the version to resolve this issue.
https://github.com/alpacahq/alpaca-py/releases/tag/v0.14.0
